pkg/flare/archive.go

```diff
@@ -175,12 +175,9 @@ func getExpVar(fb flarehelpers.FlareBuilder) error {
 		}
 	}
 
-	apmPort := "8126"
-	if config.Datadog.IsSet("apm_config.receiver_port") {
-		apmPort = config.Datadog.GetString("apm_config.receiver_port")
-	}
+	apmDebugPort := config.Datadog.GetInt("apm_config.debug.port")
```

Comment:
I'm assuming there's no default value we can fall back to, like 8126 previously?